src/net_processing.cpp

```diff
@@ -3019,6 +3019,15 @@ void PeerManager::ProcessMessage(CNode& pfrom, const std::string& msg_type, CDat
             if (!fRejectedParents) {
                 const auto current_time = GetTime<std::chrono::microseconds>();
 
+                // To save bandwidth during IBD, stop orphan processing early. With the state of
+                // our UTXO set far from tip, we might not have yet the unspent output spend by this
+                // transaction or this transaction is already double-spend but not yet seen block.
+                // Further, we can't assert accuracy of its feerate.
+                //
+                // Orphan transaction is discarded. Note, we don't both to add id to recentRejects
+                // as it's expected to be quickly clean up during IBD.
+                if (ChainstateActive().IsInitialBlockDownload()) return;
```

Comment:
d65dca43 partial suggestion
```diff
-                // To save bandwidth during IBD, stop orphan processing early. With the state of
-                // our UTXO set far from tip, we might not have yet the unspent output spend by this
-                // transaction or this transaction is already double-spend but not yet seen block.
-                // Further, we can't assert accuracy of its feerate.
+                // If we are in IBD, exit from orphan processing early to save bandwidth. With the
+                // state of our UTXO set far from the tip, we might not yet have the unspent output
+                // of this transaction, or the transaction is already double-spent but not yet
+                // included in a block. We also cannot assert the accuracy of its feerate.
                 //
-                // Orphan transaction is discarded. Note, we don't both to add id to recentRejects
-                // as it's expected to be quickly clean up during IBD.
-                if (ChainstateActive().IsInitialBlockDownload()) return;
+                // Orphan transaction is discarded. We don't bother to add it to recentRejects, as
+                // it's expected to be quickly cleaned up during IBD.
+                if (::ChainstateActive().IsInitialBlockDownload()) return;
```
"it's expected" is somewhat ambiguous, could you use an explicit subject?

Reply:
Yes this is the recentRejects filter. I did modify the comment to underscore that during IBD blocks are received fast and thus the filter quickly cleanup in consequence.